libcxx/utils/libcxx/test/format.py

```diff
@@ -52,6 +52,21 @@ def _executeScriptInternal(test, litConfig, commands):
     return (out, err, exitCode, timeoutInfo, parsedCommands)
 
 
+def _validateModuleDependencies(modules):
+    for m in modules:
+        if m not in ("std", "std.compat"):
+            raise RuntimeError(
+                f"Invalid module dependency '{m}', only 'std' and 'std.compat' are valid"
+            )
+
+
+def _getSubstitution(substitution, config):
```

Comment:
Suggestion: Use `dsl._getSubstitution` instead. It's kind of an implementation detail of that other file, but whatever.

Reply:
This doesn't work, dsl depends on format. What would work is move this to config. I've copied the _appendToSubstitution and will clean up in a follow-up commit.